support --log / -l for invoke local (especially for Python logging)

# Feature Proposal

## Description

Currently Python logging doesn't output anything using invoke local

Add logging support similar to `invoke -l`

Especially with Python logging

I'd argue that --log for invoke local should be enabled by default, rather than needing to be enabled explicitly.

## Additional Data

* ***Serverless Framework Version you're using***: 1.20.2
* ***Operating System***: Windows 10

I don't quite follow what you're asking for. `sls invoke local` already shows `stderr` and `stdout`, unlike `sls invoke` so there's no need to add a `--log` option AFAICT.

Here's the `aws-python3` template with `print('foo')` added:
```
$ sls invoke local -f hello
foo
{
    "statusCode": 200,
    "body": "{\"message\": \"Go Serverless v1.0! Your function executed successfully!\", \"input\": {}}"
}
$ sls invoke -f hello
{
    "statusCode": 200,
    "body": "{\"message\": \"Go Serverless v1.0! Your function executed successfully!\", \"input\": {}}"
}
$ sls invoke -l -f hello
{
    "statusCode": 200,
    "body": "{\"message\": \"Go Serverless v1.0! Your function executed successfully!\", \"input\": {}}"
}
--------------------------------------------------------------------
START RequestId: 1e2d9cd9-8e90-11e7-bf98-a994067d9d91 Version: $LATEST
foo
END RequestId: 1e2d9cd9-8e90-11e7-bf98-a994067d9d91
REPORT RequestId: 1e2d9cd9-8e90-11e7-bf98-a994067d9d91  Duration: 0.39 ms       Billed Duration: 100 ms    Memory
 Size: 1024 MB    Max Memory Used: 21 MB
```

and using the python `logging` module:
```
[:wat] $ sls invoke local -f hello
ERROR:root:foo

{
    "statusCode": 200,
    "body": "{\"message\": \"Go Serverless v1.0! Your function executed successfully!\", \"input\": {}}"
}
$ sls invoke -f hello
{
    "statusCode": 200,
    "body": "{\"message\": \"Go Serverless v1.0! Your function executed successfully!\", \"input\": {}}"
}
$ sls invoke -l -f hello
{
    "statusCode": 200,
    "body": "{\"message\": \"Go Serverless v1.0! Your function executed successfully!\", \"input\": {}}"
}
--------------------------------------------------------------------
START RequestId: aa1afece-8e90-11e7-a0a7-fbe2e186fe49 Version: $LATEST
2017-08-31 17:09:23.822 (-04:00)        aa1afece-8e90-11e7-a0a7-fbe2e186fe49    [ERROR] foo
END RequestId: aa1afece-8e90-11e7-a0a7-fbe2e186fe49
REPORT RequestId: aa1afece-8e90-11e7-a0a7-fbe2e186fe49  Duration: 1.02 ms       Billed Duration: 100 ms    
emory Size: 1024 MB     Max Memory Used: 21 MB
```

--- a/lib/plugins/aws/invokeLocal/invoke.py
+++ b/lib/plugins/aws/invokeLocal/invoke.py
@@ -1,5 +1,6 @@
 import argparse
 import json
+import logging
 import sys
 from time import time
 from importlib import import_module
@@ -36,6 +37,8 @@ class FakeLambdaContext(object):
     def aws_request_id(self):
         return '1234567890'
 
+
+logging.basicConfig()
 
 parser = argparse.ArgumentParser(
     prog='invoke',

